File: src/net/merayen/elastic/uinodes/list/midi_spread_1/Info.java
package net.merayen.elastic.uinodes.list.midi_spread_1;

import net.merayen.elastic.uinodes.BaseInfo;

public class Info implements BaseInfo {
	@Override public String getName() {return "Midi Spread";}
	@Override public String getDescription() {return "Spreads midi notes, making sound fuller";}
	@Override public String[] getCategories() {return new String[]{"Tools"};}
}

File: src/main/kotlin/net/merayen/elastic/uinodes/list/sample_1/Info.kt
package net.merayen.elastic.uinodes.list.sample_1

import net.merayen.elastic.uinodes.BaseInfo

class Info : BaseInfo {
	override fun getName() = "Sample"
	override fun getDescription() = "Samples sound"
	override fun getCategories() = arrayOf("Audio")
}